src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs

```diff
@@ -194,7 +200,102 @@ public void SetupEnvironment(AzureModule mode)
 #endif
         }
 
-        private void SetupAzureEnvironmentFromEnvironmentVariables(AzureModule mode)
+        public void SetEnvironmentVariableFromCredentialFile()
+        {
+            var filePath = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) + "/.azure/testcredentials.json";
```

Comment:
use System.IO.Path.Join, instead of string concatenation - it does a better job of parsing off trailing slashes, and is platform-agnostic